api/tests/integration/ref/formats/ket_to_mol.py.out

```diff
@@ -20,3 +20,4 @@ suplabel.ket:SUCCEED
 thymine.ket:SUCCEED
 empty_apid.ket:SUCCEED
 ket-reaction-arrow.ket:SUCCEED
+sgroups_mul.ket:SUCCEED
```

Comment:
Please move this line between "rna_mod.ket:SUCCEED" and "suplabel.ket:SUCCEED" - as you can see in ket_to_mol.py:50 filenames sorted before use.